.github/workflows/pytest.yml

```diff
@@ -0,0 +1,20 @@
+name: PyTest
+on: pull_request
+jobs:
+  pytest:
+    name: Run the application's tests with PyTest
+    runs-on: ubuntu-20.04
+    container: fedora:34
+    steps:
+      - name: Install Pipenv and git
+        run: dnf install -y pipenv git
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup Python
```

Comment:
What is this step needed for?

Reply:
Tried few actions to make it work.
Do you think this is the cause of the check failure?

Reply:
It's from yesterday's presentation about installing PyTest in our project.
![image](https://user-images.githubusercontent.com/88926592/159247246-6690ba95-7a3c-450d-a9b3-3c1631249867.png)

Reply:
I'm asking about `name: Setup Python` specifically. I don't see it in the presentation either.

Reply:
> I'm asking about `name: Setup Python` specifically. I don't see it in the presentation either.
You're right. Added it to make things work after few failures.
I'll remove it, as I think it doesn't make any difference.